### PR TITLE
sessionctx/variable: refine slow log output by not print useless info

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed and how it works

No need to print cop info when there is no cop task.

eg:

```sql
# Time: 2019-09-03T22:29:15.928961+08:00
# Txn_start_ts: 0
# User: root@::1
# Conn_ID: 1
# Query_time: 4.55850739
# DB: test
# Is_internal: false
# Digest: ff607ecdf92ded60eae3d47bd165392258321403520610cf5354fd5cf7057c53
# Num_cop_tasks: 0
# Cop_proc_avg: 0 Cop_proc_p90: 0 Cop_proc_max: 0 Cop_proc_addr:
# Cop_wait_avg: 0 Cop_wait_p90: 0 Cop_wait_max: 0 Cop_wait_addr:
# Succ: true
split table t between (0) and (10000) regions 5;
```

As you can see, `Num_cop_tasks` is 0, then the cop log is redundant.

### CODE REVIEW

**Comment by shenli** [comment]
Could we keep the `# Num_cop_tasks: 0`?

**Comment by crazycs520** [comment]
@shenli, Of course,  Done.

**Comment by crazycs520** [comment]
@qw4990 PTAL

File: sessionctx/variable/session.go [hunk 1]
```diff
@@ -1140,16 +1140,18 @@ func (s *SessionVars) SlowLogFormat(logItems *SlowQueryLogItems) string {
...
-			SlowLogCopProcAvg, SlowLogSpaceMarkStr, logItems.CopTasks.AvgProcessTime.Seconds(),
-			SlowLogCopProcP90, SlowLogSpaceMarkStr, logItems.CopTasks.P90ProcessTime.Seconds(),
-			SlowLogCopProcMax, SlowLogSpaceMarkStr, logItems.CopTasks.MaxProcessTime.Seconds(),
-			SlowLogCopProcAddr, SlowLogSpaceMarkStr, logItems.CopTasks.MaxProcessAddress) + "\n")
-		buf.WriteString(SlowLogRowPrefixStr + fmt.Sprintf("%v%v%v %v%v%v %v%v%v %v%v%v",
-			SlowLogCopWaitAvg, SlowLogSpaceMarkStr, logItems.CopTasks.AvgWaitTime.Seconds(),
-			SlowLogCopWaitP90, SlowLogSpaceMarkStr, logItems.CopTasks.P90WaitTime.Seconds(),
-			SlowLogCopWaitMax, SlowLogSpaceMarkStr, logItems.CopTasks.MaxWaitTime.Seconds(),
-			SlowLogCopWaitAddr, SlowLogSpaceMarkStr, logItems.CopTasks.MaxWaitAddress) + "\n")
+		if logItems.CopTasks.NumCopTasks > 0 {
```
Comment (zz-jason) [inline sessionctx/variable/session.go]: if `NumCopTasks == 1`, we only need to print `SlowLogCopProcAvg`

Comment (crazycs520) [reply sessionctx/variable/session.go]: `Cop_wait_avg`, `cop_proc_addr` also need?
done.

**Pull request merged**